py-polars/polars/series/series.py

```diff
@@ -350,7 +350,7 @@ def __init__(
         elif _check_for_pandas(values) and isinstance(
             values, (pd.Series, pd.Index, pd.DatetimeIndex)
         ):
-            self._s = pandas_to_pyseries(name, values, dtype=dtype)
+            self._s = pandas_to_pyseries(name, values, dtype=dtype, strict=strict)
```

Comment:
can we pass this to `arrow_to_pyseries` a few lines above too? and include a test for that case?